src/components/dialog.js

```diff
@@ -504,14 +504,23 @@ export class Dialog {
           }
           setImportantStyles(this.getElement(), immediateStyles);
 
-          return transition(
-            this.getElement(),
-            {
-              'transform': this.getDefaultTranslateY_(),
-            },
-            300,
-            'ease-out'
-          );
+          const executeTransition = () => {
+            transition(
+              this.getElement(),
+              {
+                'transform': this.getDefaultTranslateY_(),
+              },
+              300,
+              'ease-out'
+            );
+          };
+          if (typeof requestAnimationFrame === 'function') {
```

Comment:
So I think this check could be removed. Swgjs' supported browsers have the `requestAnimationFrame` method

Reply:
Oh cool, good to know. Reverted the last commit.

Reply:
Thanks!